geotrek/core/serializers.py

```diff
@@ -32,6 +32,7 @@ class Meta(MapentityGeojsonModelSerializer.Meta):
 class TrailSerializer(DynamicFieldsMixin, serializers.ModelSerializer):
     length = serializers.FloatField(source='length_display')
     name = serializers.CharField(source='name_display')
+    certifications = serializers.CharField(source='certifications_display')
```

Comment:
if you add certification info in serializer,  add select_related or prefetch_related in queryset

Reply:
Yes ! I did it in `TrailFormatList`